cmd/govim/internal/vimconfig/vimconfig.go

```diff
@@ -87,6 +89,25 @@ func copyStringValMap(i, j *map[string]string) *map[string]string {
 	return &res
 }
 
+func copyBoolValMap(i *map[string]int, j *map[string]bool) *map[string]bool {
```

Comment:
I think this logic needs to be a merge, first applying `j`, then `i`, no? Otherwise if I supply a non-nil `i` then non of the defaults in `j` are applied.

Reply:
That was my initial though as well, then I noticed that the others (`copyStringValMap` and `copyMap`) didn't. Or I might have misunderstood `ToConfig(...)`, hm.

`j` are defaults , so in case of conflicting keys the ones in `i` should still override right?

Reply:
> then I noticed that the others (copyStringValMap and copyMap) didn't. Or I might have misunderstood ToConfig(...), hm.

I think that's just a case of neither the experimental popup options nor the env maps currently having defaults. If they did then we would need a merge. So more a case of me being lazy/writing for the situation at the time than anything. We could equally tidy them up in the interests of completeness though, you're right.

> j are defaults , so in case of conflicting keys the ones in i should still override right?

Exactly.